kubejs/server_scripts/base/recipes/minecraft/shaped.js

```diff
@@ -9,6 +9,26 @@ onEvent('recipes', (event) => {
                 A: '#minecraft:logs'
             },
             id: `${id_prefex}stick_16`
+        },
+        {
+            output: '4x chest',
+            pattern: ['SSS',
+                      'S S',
+                      'SSS'],
+            key: {
+                S: '#minecraft:logs'
+            },
+            id: `${id_prefex}chest_4`
+        },
+        {
+            output: '1x chest',

```

Comment:
This recipe will sadly override all chests for specific materials, such as Jungle/Spruce.

Reply:
There's already a mod that allows us to switch the result though, I have tested it and I can still choose between the normal chest or the specific chest